packages/active-element/README.md

```diff
@@ -0,0 +1,86 @@
+# @solid-primitives/active-element
+
+[![lerna](https://img.shields.io/badge/maintained%20with-lerna-cc00ff.svg?style=for-the-badge)](https://lerna.js.org/)
+[![size](https://img.shields.io/bundlephobia/minzip/@solid-primitives/active-element?style=for-the-badge)](https://bundlephobia.com/package/@solid-primitives/active-element)
+[![size](https://img.shields.io/npm/v/@solid-primitives/active-element?style=for-the-badge)](https://www.npmjs.com/package/@solid-primitives/active-element)
+
+- [`createActiveElement`](#createActiveElement) - A reactive `document.activeElement`. Check which element is currently focused.
```

Comment:
I'm trying to move the docs away from listing the primitives at the top and treating the how to use as a way to list them with a description. Basically createActiveElement and createIsElementActive can be an h3 header within how to use.

Reply:
Yeah, I saw that, but I like the listing because it lets put links to specific sections, making it work as a table of contents of sorts. And it's easier to scan that small section alone, than having to look through the whole readme.

Reply:
Fair enough. I did remove it from the other packages but I'll think of a nicer way to manage this.